lib/fluent/plugin/out_influxdb.rb

```diff
@@ -56,6 +61,7 @@ def initialize
   end
 
   def configure(conf)
+    compat_parameters_convert(conf, :buffer)
     super
   end
```

Comment:
Does this plugin always needs `tag`? If so, raise configuration error here if `@chunk_key_tag` is false.